Firefox-location returns an object

```bash
C:\Users\Michael\Github\supported-audio>node
> require('firefox-location')
{}
> require('child_process').spawn(require('firefox-location'), [ 'http://josephdykstra.com' ])
TypeError: Bad argument
    at TypeError (native)
    at ChildProcess.spawn (child_process.js:1136:26)
    at Object.exports.spawn (child_process.js:995:9)
    at repl:1:26
    at REPLServer.defaultEval (repl.js:132:27)
    at bound (domain.js:254:14)
    at REPLServer.runBound [as eval] (domain.js:267:12)
    at REPLServer.<anonymous> (repl.js:279:12)
    at REPLServer.emit (events.js:107:17)
    at REPLServer.Interface._onLine (readline.js:214:10)
>
```

I can replicate this issue using firefox-launch.

```bash
C:\Users\Michael\Github\supported-audio>node
> require('firefox-launch')('http://josephdykstra.com')
TypeError: Bad argument
    at TypeError (native)
    at ChildProcess.spawn (child_process.js:1136:26)
    at exports.spawn (child_process.js:995:9)
    at launchFirefox (C:\Users\Michael\Github\supported-audio\node_modules\firefox-launch\index.js:38:12)
    at repl:1:26
    at REPLServer.defaultEval (repl.js:132:27)
    at bound (domain.js:254:14)
    at REPLServer.runBound [as eval] (domain.js:267:12)
    at REPLServer.<anonymous> (repl.js:279:12)
    at REPLServer.emit (events.js:107:17)
>
```

I originally ran into this issue using smokestack...

```bash
C:\Users\Michael\Github\supported-audio>npm t

> supported-audio@1.0.1 test C:\Users\Michael\Github\supported-audio
> browserify test.js | tap-closer | smokestack -b firefox

child_process.js:1136
  var err = this._handle.spawn(options);
                         ^
TypeError: Bad argument
    at TypeError (native)
    at ChildProcess.spawn (child_process.js:1136:26)
    at exports.spawn (child_process.js:995:9)
    at launchFirefox (C:\Users\Michael\Github\supported-audio\node_modules\smokestack\node_modules\firefox-launch\index.js:38:12)
    at launchBrowser (C:\Users\Michael\Github\supported-audio\node_modules\smokestack\index.js:188:20)
    at open (C:\Users\Michael\Github\supported-audio\node_modules\smokestack\index.js:131:9)
    at DestroyableTransform.flush [as _flush] (C:\Users\Michael\Github\supported-audio\node_modules\smokestack\index.js:125:24)
    at DestroyableTransform.<anonymous> (C:\Users\Michael\Github\supported-audio\node_modules\smokestack\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:135:12)
    at DestroyableTransform.g (events.js:199:16)
    at DestroyableTransform.emit (events.js:129:20)
npm ERR! Test failed.  See above for more details.
```

I believe I am getting [this error on travis](https://travis-ci.org/ArtskydJ/supported-audio/jobs/79671749) as well.

--- a/index.js
+++ b/index.js
@@ -19,20 +19,13 @@ if (osx) {
     ? regPath
     : altPath
 } else {
-  var suffix = '\\Mozilla Firefox\\firefox.exe';
-  var prefixes = [
+  var suffix = '\\Mozilla Firefox\\firefox.exe'
+  module.exports = [
       process.env.LOCALAPPDATA
     , process.env.PROGRAMFILES
     , process.env['PROGRAMFILES(X86)']
-  ]
-
-  for (var i = 0; i < prefixes.length; i++) {
-    var exe = prefixes[i] + suffix
-    if (fs.existsSync(exe)) {
-      module.exports = exe
-      break
-    }
-  }
-
-  module.exports = module.exports || null
+  ].reduce(function (existing, prefix) {
+    var exe = prefix + suffix
+    return existing || fs.existsSync(exe) ? exe : null
+  }, null)
 }

